Blaise-Shyaka/AnnounceIT
# Ft signin page #170772322

# What does this PR do?

This page allows the user to sign in to his/her account

# Description of tasks to be completed

- Create a simple form and make it responsive
- The form fields should collect at least the email and the password of the user
- Provide an option of changing the password in case it was forgotten

# How should this be manually tested?

- Clone this repository and cd into /templates/signin.html
- open the file
- Resize the browser to check its responsiveness
- Check the presence of form fields where a user can fill the email and password
- Check the presence of a link to help users who have forgotten the password
